.github/workflows/old-backports

```diff
@@ -0,0 +1,21 @@
+name: Alert on Abandoned Backports
+on:
+  schedule:
+    - cron: '*/10 * * * 4'
```

Comment:
this will be `At every 10th minute on Thursday.` https://crontab.guru/#*/10_*_*_*_4

is it really intended to run that much on just thursday?

Reply:
Yep - just as a test for now. It's also configured to send messages directly to me.